### ISSUE TITLE
Stop scrape and download results

### ISSUE THREAD

**Issue by karcarkar** [arc298/instagram-scraper#557]
I have been running instagram-scraper for a few days and have a few thousand posts. As much as I would like all posts associated with the hashtag I'm scrapping, I need to start working on this data set tomorrow. Is it possible to quit the scrape and download the results that I have so far?

**Comment by aboutqx** [comment]
Just `ctrl c`. See the destination folder, downloaded images will stay there, not get deleted.

**Comment by karcarkar** [comment]
Yes, I've got the images - but what about the json file? My scrape has currently stalled - I have a folder that has downloaded roughly half the images, but no json, which is what I really need.

**Comment by rizkyekoputra** [comment]
the same problem here, any way to skip the download part, I just need the .json part
